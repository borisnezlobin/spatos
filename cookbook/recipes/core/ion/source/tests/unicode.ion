fn Җ Ҕ:int
  echo $Ҕ
end

echo Ҥ
Җ 1
let Ҳ = ӌ
echo $Ҳ
let 1Ҳ = ӌ
echo $1Ҳ
echo Should not be reached
