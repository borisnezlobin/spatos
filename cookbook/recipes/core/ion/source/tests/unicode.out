Ҥ
1
ӌ
ion: assignment error: invalid variable name: only alphanumerical characters and underscores are supported
ion: expansion error: Variable "1Ҳ" does not exist
